src/styled/StyledHeader.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const StyledHeader = styled.div`
```

Comment:
```suggestion
const StyledHeader = styled.header`
```
Since this is the header, you should use the `header` tag.